### DISCUSSION TITLE
Feature request: way to track animals that enter with low survival chances without creating a new animal record

### DISCUSSION THREAD

**Discussion by Pletoo** [sheltermanager/asm3#1533] [Ideas]
We often get situations like:

- Picked up injured cat near highway
- Found kitten dropped by eagle
- Litter of kittens left at campground, no mother (maybe 3 weeks old?)
- Feral cat we just brought in yesterday had kittens overnight

Unfortunately these cats and kittens don't have a high survival rate but we still need to track them. 
They don't often last long enough to get to a vet or to have a SM record created for them.
Currently I am using a spreadsheet that I print separately in addition to reports from SM when we are looking at inventory and statistics.
It includes: date of entry, location of pickup, comment, date of death.

It would be helpful to be able to track these within SM without having to create an individual record for each of them, perhaps as a log or a table somewhere, and have that data accessible, especially when we are trying to identify trouble spots.

**Comment by bobintetley** [comment]
Hi, this is what the DOA flag on the death slider is intended for. It still requires an animal record though I'm afraid. I'm not sure it makes sense to make a whole new table and reports to do what you can do already with an animal record.
